assets/sass/_jetpack.scss

```diff
@@ -222,7 +222,7 @@ Featured Content
 			right: 0;
 		}
 
-		@media screen and (max-width: $width-medium) {
+		@include max-width($width-medium) {
```

Comment:
Expected item on line 225 to appear before line 217. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets